source/features/rgh-linkify-features.tsx

```diff
@@ -5,13 +5,27 @@ import * as pageDetect from 'github-url-detection';
 import {wrap} from '../helpers/dom-utils';
 import features from '.';
 import {getNewFeatureName} from '../options-storage';
-import {isNotRefinedGitHubRepo} from '../github-helpers';
+import {isRefinedGitHubRepo} from '../github-helpers';
 import onConversationHeaderUpdate from '../github-events/on-conversation-header-update';
 
 function linkifyFeature(codeElement: HTMLElement): void {
 	const id = getNewFeatureName(codeElement.textContent!) as FeatureID;
-	if (features.list.includes(id) && !codeElement.closest('a')) {
-		wrap(codeElement, <a href={`/refined-github/refined-github/blob/main/source/features/${id}.tsx`}/>);
+	if (features.list.includes(id)) {
+		const href = `/refined-github/refined-github/blob/main/source/features/${id}.tsx`;
+
+		const possibleLink = codeElement.firstElementChild ?? codeElement;
+		if (possibleLink instanceof HTMLAnchorElement) {
```

Comment:
I'm concerned about this logic. How are we sure that the possibleLink isn't the whole line instead of just the feature name?

I'd like to extract this part into a function that accepts a `code` element and linkifies it the best way it can.

Reply:
Good call. I'll add a extra check

Reply:
The code would still benefit from some clarification. The variable name is codeElement but now it seems that it can also be a link. 

Can you add a comment before each `if` explaining the expected DOM?